report.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+echo "Enter cluster name (default my-cluster)"
+read cluster
```

Comment:
Should the script support `--namespace` and `--cluster` options, and only prompt if they're not present on the command line?